app/models/project.rb

```diff
@@ -1,3 +1,13 @@
 class Project < ApplicationRecord
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def theme
+    self.challenge.theme
+  end
+
+  def contestant_count
+    self.contestants.count
+  end
```

Comment:
You have these two methods written, but you didn't use them in your view. This is exactly what I described in my other comment, so you'd want to use these model methods in your view to call `project.contestant_count`